src/Ombi/ClientApp/app/settings/themoviedb/themoviedb.component.html

```diff
@@ -0,0 +1,29 @@
+﻿<settings-menu></settings-menu>
+
+<fieldset *ngIf="settings">
+    <legend>The Movie Database</legend>
+    <div class="col-md-6">
+        <div class="form-group">
+            <div class="checkbox">
+                <input type="checkbox" id="showAdultMovies" name="showAdultMovies" [(ngModel)]="settings.showAdultMovies">
+                <label for="showAdultMovies" tooltipPosition="top" pTooltip="Include adult movies (pornography) in results">Show Adult Movies</label>
+            </div>
+        </div>
+
+        <div class="form-group">
+            <label for="excludedKeywordIds" class="control-label" pTooltip="Prevent movies with certain keywords from being suggested. May require a restart to take effect.">
+                Excluded Keyword IDs for Movie Suggestions
+            </label>
+            <div>
+                <input type="text" [(ngModel)]="settings.excludedKeywordIds" class="form-control form-control-custom " id="excludedKeywordIds"
```

Comment:
Might be worth using some sort of tag box instead here, for example:
https://www.primefaces.org/primeng/#/chips

Reply:
We could also use the Keyword search API so the user can easily lookup keywords instead of attempting to find the ID's?
https://developers.themoviedb.org/3/search/search-keywords

What do you think?

Reply:
@tidusjar I generally agree, as that was my original thought. I was mostly just trying to get out a working filter so I could get the specified movie off my home page since my parents and other family use Ombi. However, I also wasn't sure about how to handle the [different separators](https://developers.themoviedb.org/3/discover/movie-discover) in this case since you can use `,` (`AND`) or `|` (`OR`). I tested with both and they seem to provide the same results for the `without_keyword` filter, but I wasn't really confident enough to just pick using one over the other. I'm okay with just picking one like I did in the `placeholder` if that's what you would prefer.

Reply:
@tidusjar A couple questions:
1. [ngx-chips](https://github.com/Gbuomprisco/ngx-chips) supports [autocomplete within their control](https://angular-mfppay.stackblitz.io/) ([unlike primeng](https://github.com/primefaces/primeng/issues/5551)). With something like that, a separate search control would be unnecessary. I know Ombi is already referencing `primeng`, but would you be opposed to me adding a reference to `ngx-chips` as well for this?
2. TMDB doesn't provide an endpoint to retrieve multiple keywords at a time by ID so we would likely still want to show the keyword ID in each chip to avoid having to send `n` queries on load. I was thinking the autocomplete could search by the keyword (`/search/keyword`) but clicking on an existing chip would show the name (`/keyword`) in a popover or something. How does that sound?

Reply:
1. Yeah that's fine, I only referenced primeng because I knew they had an existing control. But if you have found a better one then I'm all for it. 
2. That sounds like a good idea.